modules/lib/maintainers.nix

```diff
@@ -215,6 +214,13 @@
       fingerprint = "8F87 050B 0F9C B841 1515  7399 B784 3F82 3355 E9B9";
     }];
   };
+  NotAShelf = {
+    name = "NotAShelf";
+    email = "raf@notashelf.dev";
+    github = "NotAShelf";
+    githubId = 62766066;
+    matrix = "@raf:notashelf.dev";
+  };
```

Comment:
```suggestion
```

You're already a nixpkgs maintainer, and you're not using this entry here anyway